apprentice-react-o-shop/src/components/NavBar/NavBar.js

```diff
@@ -14,20 +14,26 @@ function NavBar(){
             <div className="collapse navbar-collapse" id="navbarsExampleDefault">
                 <ul className="navbar-nav mr-auto">
                 <li className="nav-item active">
-                    <a className="nav-link" href="/#">Home <span className="sr-only">(current)</span></a>
+                    <a className="nav-link" href="/">Home <span className="sr-only">(current)</span></a>
```

Comment:
How can we refactor this to not have so much repetition

Reply:
I added two components that take a link and text as props for the main bar and the dropdown.